conf/modules.config

```diff
@@ -238,6 +238,38 @@ process {
         ].join(' ').trim() }
     }
 
+    withName: LIMMA_NORM {
+        ext.prefix = 'all'
+        publishDir = [
+            [
+                path: { "${params.outdir}/tables/processed_abundance" },
+                mode: params.publish_dir_mode,
+                pattern: '*.{normalised_counts}.tsv'
+            ],
+        ]
+        ext.when = params.limma_use_voom
+        ext.args = { [
+            "--probe_id_col \"${params.features_id_col}\"",
+            "--sample_id_col \"${params.observations_id_col}\"",
+            "--ndups \"${params.limma_ndups}\"",
+            "--spacing \"${params.limma_spacing}\"",
+            "--block \"${params.limma_block}\"",
+            "--correlation ${params.limma_correlation}",
+            "--method \"${params.limma_method}\"",
+            "--proportion ${params.limma_proportion}",
+            "--stdev.coef.lim \"${params.limma_stdev_coef_lim}\"",
+            "--trend ${params.limma_trend}",
+            "--robust ${params.limma_robust}",
+            "--winsor.tail.p \"${params.limma_winsor_tail_p}\"",
+            "--p.value ${params.limma_p_value}",
+            "--lfc ${params.limma_lfc}",
+            "--confint ${params.limma_confint}",
+            "--subset_to_contrast_samples $params.differential_subset_to_contrast_samples",
```

Comment:
Don't pass this to LIMMA_NORM, the idea is to avoid doing anything that messes with the matrix. We should take it out of DESEQ_NORM as well, I don't remember why it's there, probably just an error.